### PR TITLE
[Localization] #1761 Korean trainer dialogue (Unova region)

### PR DESCRIPTION
**Pull request by sodaMelon**
## What are the changes?

Translating locales\ko\dialogue.ts into Korean.
This PR is added korean dialogue for **named trainers in Hana  region.** (5th Pokemon Games)

@sodaMelon  @returntoice now participating in https://github.com/pagefaultgames/pokerogue/issues/1761 translating project.
@ Ohmry said he never played original Pokemon Game. so I decide to exclude this time.
(he only played Legend Arceus)

## Why am I doing these changes?

A few trainers's dialogue have been translated into Korean.
I did the others.

## What did change?

I translated **Gym leader,Elite-4 and Champion(in Hana, 5th pokemon)battles' dialogues.**
5세대 체육관 관장+사천왕 + 아이리스(챔피언) 다이얼로그를 번역했습니다.)

말투 교정에는 정발판 5세대 포켓몬스터 블랙&화이트1, 블랙&화이트2 참고했습니다.

(팟, 덴트, 콘은 누가 하셨더라구요? 그래서 그 친구들은 안했습니다.)

### Screenshots/Videos

( very hard to meet them.... and it's dosen't matter)

## How to test the changes?

It seems there won't be much to test since it's just a translation of an existing .ts file. You can check it when you meet the respective trainers.

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- ~~[ ] Are the changes visual?~~
  - ~~[ ] Have I provided screenshots/videos of the changes?~~

### CODE REVIEW

**Review by EnochG1 — COMMENTED** [review]
I suggested minor things. If there's reason, you can skip :)

File: src/locales/ko/dialogue.ts [hunk 1]
```diff
@@ -1950,68 +1950,68 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
-      1: "That was unsatisfying somehow… Will you give it your all next time?"
+      1: "왠지 부족한 기분이야… 다음에는 최선을 다할 수 있지?"
     }
   },
   "skyla": {
     "encounter": {
-      1: `It's finally time for a showdown! That means the Pokémon battle that decides who's at the top, right? 
-                    $I love being on the summit! 'Cause you can see forever and ever from high places! 
-                    $So, how about you and I have some fun?`,
+      1: `드디어 대결의 시간이야! 누가 정상에 오를지 결정짓는 포켓몬 배틀-, 그치?
```
[the remark this answers is not in the document]
Comment (sodaMelon) [reply src/locales/ko/dialogue.ts:1982]: 이 친구 대사(인게임 스크린샷확인하시면 원래 하이픈 넣어서 얘기합니다

**Comment by sodaMelon** [comment]
@EnochG1 thanks to detail review. I collapsed once this week due to health problems, so little less accurate.
Your suggestion was applied. Please Check when you have a free time

**Comment by sodaMelon** [comment]
I change Hana -> Unova
I was confused with the Japanese version of Localizing

File: src/locales/ko/dialogue.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (sodaMelon) [reply src/locales/ko/dialogue.ts:1982]: 아니 다

**Review by Tempo-anon — APPROVED** [review]
Please take care of your health first and foremost sodaMelon

**Pull request merged**